vendor/github.com/mongodb/mongo-tools-common/db/namespaces.go

```diff
@@ -45,48 +43,13 @@ func (ci *CollectionInfo) GetUUID() string {
 			if x.Subtype == 4 {
 				return hex.EncodeToString(x.Data)
 			}
-		case bson.Binary:
```

Comment:
why was this removed?

Reply:
`bson.Binary` is an mgo type, `primitive.Binary` is the go driver one. this function apparently handled both (you can see if you expand the diff), so I just removed the mgo one.